docs/sources/whatsnew/whats-new-next/index.md

```diff
@@ -18,6 +18,60 @@ weight: -37
 
 Welcome to Grafana Cloud! Read on to learn about the newest changes to Grafana Cloud.
 
+## User experience updates in Kubernetes Monitoring
+
+<!-- Beverly Buchanan --> 
+
+_Generally available in Grafana Cloud_
+
+### Quickly troubleshoot with snapshot of issues
+
+Release 1.3.1. To provide quicker identification and troubleshooting, you can view a snapshot of issues on the home page that reach these specific thresholds:
+
+  - Pods that have been in a non-running state for more than 15 minutes
+  - Nodes with CPU usage above 90% for more than five minutes
+  - Nodes using more than 90% of memory for more than five minutes
+  - Persistent Volumes with capacity above 90%
+  - Node disks with capacity above 90%
+
+![Kubernetes Monitoring home page](/media/docs/grafana-cloud/k8s/K8smon-snapshotview.png)
+
+### Navigate with tabs
+
+Release 1.3.0. Quickly switch between the Cluster, namespace, workload, and Node views on the **Cluster Navigation** page by using tabs.
+
+![Tabs on Cluster navigation page](/media/docs/grafana-cloud/k8s/k8smon-clusternav-tabs.png)
```

Comment:
This image is really big for what it's showing. Suggest using a `[figure` shortcode](https://grafana.com/docs/writers-toolkit/write/shortcodes/#example-5) to constrain its width (see second example at this link).

Reply:
Thx! Resized